src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -529,6 +529,27 @@ public static async ValueTask<FormulaValue> SortTable(EvalVisitor runner, EvalVi
             {
                 return CommonErrors.RuntimeTypeMismatch(irContext);
             }
+        }
+
+        public static async ValueTask<FormulaValue> AsType(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
+        {
+            var arg0 = (RecordValue)args[0];

```

Comment:
did pipeline confirm they really are these types, or could they be Blank/Error?

Reply:
Yes, add a test in DV repo for the same.